agent/rpm/pbench-agent.spec

```diff
@@ -0,0 +1,127 @@
+Name:           pbench-agent
+Version:     0.60
+%define gdist gc9e89f8f
+Release:     1%{?gdist}%{!?gdist:}
+Summary:        The pbench harness
+
+License:        GPLv3+
+URL:            http://perf1.perf.lab.eng.bos.redhat.com/atheurer/pbench
+Source0:        pbench-agent-%{version}.tar.gz
+Buildarch:      noarch
+
+
+%if 0%{?rhel} == 6
+%define turbostatpkg cpupowerutils
+%else
+%define turbostatpkg kernel-tools
+%endif
+
+%if 0%{?rhel} == 7
+Requires:  scl-utils, rh-python36
+%endif
+
+%if 0%{?rhel} == 8 || 0%{?fedora} >= 29
+Requires:  python3-pip
+%endif
+
+%if 0%{?fedora} == 0
+# NOT fedora
+%define perljsonxs pbench-perl-JSON-XS
+%else
+%define perljsonxs perl-JSON-XS
+%endif
+
+
+Requires:  bzip2, tar, xz, screen
+Requires:  perl, perl-JSON, %{perljsonxs}, perl-Time-HiRes, perl-Data-UUID
+Requires:  net-tools, numactl, perf, psmisc, bc, sos, %{turbostatpkg}
```

Comment:
It seem `net-tools`, `numactl`, `perf`, and `%{turbostatpkg}` are for tools, which we need to remove from the agent dependency, but `sos`, `bc`, and `psmisc` are required for agent functioning.

We'll have to make sure the datalogs for those tools have the proper checks in them.